dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventOperationParamsMapper.java

```diff
@@ -306,8 +305,12 @@ private String getOrgUnitModeViolation(EventOperationParams params, User user, P
       case ALL -> userCanSearchOrgUnitModeALL(user)
           ? null
           : "Current user is not authorized to query across all organisation units";
```

Comment:
This validation altogether couldn't be moved to EventRequestParamsMapper?
For most of the modes it is just checking existence or not of orgUnits, for ALL mode we are checking the user and it shouldn't access the DB

Reply:
`ACCESSIBLE` and `CAPTURE` need the user to validate the org unit mode, and I wanted to keep all the unit mode validations together, that's the reason why they are in this mapper now.